habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/BuildDeploymentArtifactsMojo.java

```diff
@@ -149,4 +168,25 @@ protected void setUpPlaceholderFileAsMavenArtifact() {
         project.getArtifact().setFile(mavenArtifactFile);
     }
 
-}
+    private void cacheWheelFiles() {
+        PoetryCommandHelper poetryHelper = createPoetryCommandHelper();
+        try{
+            File wheelSourceDirectory = new File(String.format("%s/dist", project.getBasedir()));
+            String poetryCacheDirectoryPath = poetryHelper.getPoetryCacheDirectoryPath();
+            File poetryWheelCacheDirectory = new File(String.format("%s/cache/repositories/wheels/%s", poetryCacheDirectoryPath, project.getArtifactId()));
+
+            if(poetryWheelCacheDirectory.exists() || poetryWheelCacheDirectory.mkdirs()){
+                List<File> wheelFiles = Stream.of(wheelSourceDirectory.listFiles())
+                                              .filter(file -> file.getAbsolutePath().endsWith(".whl"))
+                                              .map(File::getAbsoluteFile)
+                                              .collect(Collectors.toList());
+                for(File file : wheelFiles){
+                    HabushuUtil.copyFile(file.getPath(), String.format("%s/%s", poetryWheelCacheDirectory.toPath().toString(), file.getName()));
+                    getLog().info(String.format("Cached the %s file", file.getName()));
```

Comment:
A: Do we always want the cached files printed or would this be a better case for getLog().debug(...)?

Reply:
<img width="806" alt="image" src="https://github.com/TechnologyBrewery/habushu/assets/6691969/fa42ed99-9ca7-4981-83b5-975bf0b71179">

This is a one liner that gets tacked onto the existing action's log output and only shows when the config is set to true. I think this follows the existing patterns within habushu.